Api/Client/HttpClientFactory.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Net.Http;
+
+namespace PetStore6.Api.Client
+{
+    public class HttpClientFactory
+    {
+        private static HttpClient _httpClient;
```

Comment:
It could be a problems with HttpClient, because you have no constructor, and it could be a null.
Declare it as a nullable
`        private static HttpClient? _httpClient;
`